DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj

```diff
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <ProjectGuid>12B1A410-DE8B-4F45-97E1-E0B502FDECC7</ProjectGuid>
-    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
This caused an issue on the pipeline but I can't recall what. Just a heads up. No action needed.

Reply:
thumbs up emoji.  It would be good if this can be utilised. Will look into it if it causes a failure